controllers/constant/odlm.go

```diff
@@ -104,6 +104,89 @@ spec:
       apicatalogmanager: {}
       operandBindInfo: {}
       operandRequest: {}
+  - name: cloud-native-postgresql
+    resources:
+      - apiVersion: batch/v1
+        kind: Job
+        name: create-postgres-license-config
+        data:
+          spec:
+            activeDeadlineSeconds: 600
+            backoffLimit: 5
+            template:
+              metadata:
+              spec:
+                affinity:
+                  nodeAffinity:
+                    requiredDuringSchedulingIgnoredDuringExecution:
+                      nodeSelectorTerms:
+                      - matchExpressions:
+                        - key: beta.kubernetes.io/arch
+                          operator: In
+                          values:
+                          - amd64
+                containers:
+                - command:
+                  - bash
+                  - -c
+                  - |
+                    cat << EOF | kubectl apply -f -
+                    apiVersion: v1
+                    kind: Secret
+                    type: Opaque
+                    metadata:
+                      name: postgresql-operator-controller-manager-config
+                    data:
+                      EDB_LICENSE_KEY: $(base64 /license_keys/edb/EDB_LICENSE_KEY | tr -d '\n')
+                    EOF
+                  image: cp.icr.io/cp/cpd/wd-postgres-license@sha256:4d8d0ecd31d04e15f757ffda74101ef5bd7fb5789db1ffcef892e961ef312ebf
```

Comment:
@horis233 Do we need to create pull secret and link it to new SA?

Reply:
Yes we need to set up imagepullsecret `ibm-entitlement-key` in the job